Sources/PointFree/TODO.swift

```diff
@@ -309,6 +309,10 @@ public func onclick<T>(javascript: StaticString) -> Attribute<T> {
   return .init("onclick", "javascript:\(javascript)")
 }
 
+public func onclick<T>(unsafeJavascript: String) -> Attribute<T> {
+  return .init("onclick", "javascript:\(unsafeJavascript)")
```

Comment:
new helper for being explicit when we are doing potentially unsafe things. without this I needed to create two separate scripts for the timestamp links, one that prevents default and one that doesnt.